_includes/configuration/dockerized-hyperlinking-service.md

```diff
@@ -0,0 +1,134 @@
+    ├── docker-entrypoint.sh
+    ├── Dockerfile
+    ├── ephox-hyperlinking.war
+    └── generate-jetty-start.sh
+    ```
+4. Navigate into the extracted folder.
+
+    ```sh
+    cd tinymce-hyperlinking-service
+    ```
+{% endif %}
+5. Set the permissions on the extracted files to executable.
+
+    ```sh
+    chmod +x *.sh
+    ```
+6. _Optional_: Edit the `http` configuration settings in `ephox-hyperlinking-docker-env.conf`. The configurable settings are in the `http` section of the file. For example:
+
+    ```
+    http {
+      max-connections = 100
+      max-connections-per-host = 10
+      max-redirects = 10
+      max-retries = 3
+      request-timeout-seconds = 10
+      trust-all-cert = false
+      websphere = {
+        use-ssl-config = true
+      }
+    }
+    ```
+    For information on the `http` configuration setting, see: [Configure server-side components - `http`]({{site.baseurl}}/enterprise/server/configure/#httpoptional).
+1. _Optional_: Update the link-checking cache configuration, as described in [Link Checker self-hosted quick setup]({{site.baseurl}}/enterprise/check-links/#linkcheckerself-hostedquicksetup).
+1. _Optional_: Configure the Enhanced Media Embed Service, as described in [Configure Enhanced Media Embed Server]({{site.baseurl}}/enterprise/embed-media/mediaembed-server-config/).
+1. _Optional_: Configure the service to use a HTTP proxy by updating `ephox-hyperlinking-docker-env.conf`. See:
+[Configure server-side components - proxy]({{site.baseurl}}/enterprise/server/configure/#proxyoptional).
+1. Create an `origins.env` file and specify the Hypertext Transfer Protocol (HTTP) and domain name of sites hosting the TinyMCE editor (`allowed-origins`). Up to 99 origins can be added without editing `ephox-hyperlinking-docker-env.conf`.
+
+    For example:
+
+    ```conf
+    ORIGIN0=[example.net](<http://example.net/>)
+    ORIGIN1=example
+    ORIGIN2=http://[example.org](<http://example.org//>)
+    ```
+    For information on `allowed-origins`, see: [Configure server-side components - allowed-origins]({{site.baseurl}}/enterprise/server/configure/#allowed-originsrequired).
+2. As the root user or Administrator, build the {{site.productname}} Hyperlinking Docker image using the following command:
+
+    ```sh
+    docker build -t tinymce-hyperlinking-service .
+    ```
+3. As the root user or Administrator, deploy the service using the following command:
+
+    ```sh
+    docker run -d -p 8083:8080 --env-file origins.env tinymce-hyperlinking-service
+    ```
+    Where:
+    * `-p 8083:8080` maps the container port `8080` to local port `8083`.
+    * `--env-file origins.env` adds the allowed origins to the container.
+4.  To verify that the Docker container is deployed and the hyperlinking service is running, execute:
+
+    ```sh
+    curl http://localhost:8083/ephox-hyperlinking/
+    ```
+    The response from the `curl` command should be:
+    ```
+    Link checking and media embedding service is running.
```

Comment:
This is rendering with syntax highlighting in the HTML, as are the other response messages in the doc, but I guess that's not a bug you can deal with here since you haven't specified a language 🤔

Reply:
Tried fixing this again, not sure which syntax highlighter the docs use, it's one of three, it's not a critical problem so won't fix.